Tatsi/Config/TatsiColors.swift

```diff
@@ -56,4 +59,12 @@ public struct TatsiDefaultColors: TatsiColors {
             return .gray
         }
     }()
+    
+    public let checkMark: UIColor = {
```

Comment:
When making `checkMark` optional, you don't have to supply this method anymore.

Reply:
To be compliant with the protocol I still need to provide a variable and the goal to be compatible with older versions is not resolved this way.